home-data-migration/src/main/java/Main.java

```diff
@@ -0,0 +1,16 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        String pass, user;
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Please, enter the username");
+        user = scanner.nextLine();
```

Comment:
Please do not use any prompts. We'll use this app in a non-tty environment so just take required data from arguments.